site/content/config.js

```diff
@@ -1,50 +1,52 @@
-const config = {
-  title: 'Flowershow',
-  description: 'Publish your digital garden',
-  author: 'Flowershow',
-  authorLogo: '/assets/images/logo.svg',
-  authorUrl: 'https://flowershow.app/',
+export const config = {
+  title: "Flowershow",
+  description: "Publish your digital garden",
+  author: "Flowershow",
+  authorLogo: "/assets/images/logo.svg",
+  authorUrl: "https://flowershow.app/",
   navbarTitle: {
-    logo: '/assets/images/logo.svg',
-    text: 'Flowershow',
-    version: 'Alpha'
+    logo: "/assets/images/logo.svg",
+    text: "Flowershow",
+    version: "Alpha",
   },
   repoRoot: "https://github.com/flowershow/flowershow",
   repoEditPath: "/edit/main/site/content/",
   editLinkShow: false,
-  analytics: 'G-RQWLTRWBS2',
+  analytics: "G-RQWLTRWBS2",
```

Comment:
I prefer `'` to `"`. Does eslint default to always requiring `"`. Again this one i don't mind and more a question of consistency i guess.